include/HubFramework/HUBViewModelLoader.h

```diff
@@ -103,6 +103,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)loadViewModel;
 
 /**
+ *  Load a view model using this loader regard less of reload policy
+ */
+- (void)loadViewModelRegardlessOfReloadPolicy;
```

Comment:
Could this method be moved to `HUBViewModelLoaderImplementation` instead? That way we don't have to make the public API more complicated, because this should really be seen as an implementation detail.